include/picotls.h

```diff
@@ -1359,7 +1359,8 @@ inline void ptls_buffer_init(ptls_buffer_t *buf, void *smallbuf, size_t smallbuf
 inline void ptls_buffer_dispose(ptls_buffer_t *buf)
 {
     ptls_buffer__release_memory(buf);
-    *buf = (ptls_buffer_t){NULL};
+    ptls_buffer_t empty_buffer = {0};
+    *buf = empty_buffer;
```

Comment:
`*buf = {0}`?

Reply:
fails to build on windows... only works for initialization, not assignment.